src/control/client/connect.go

```diff
@@ -302,6 +302,9 @@ type Connect interface {
 	PoolDestroy(*PoolDestroyReq) error
 	BioHealthQuery(*mgmtpb.BioHealthReq) ResultQueryMap
 	SmdListDevs(*mgmtpb.SmdDevReq) ResultSmdMap
+// this is just a guess.  I'm not sure how these two functions should be defined
+	RequestProviderList(*ctlpb.ProviderListRequest) (*ctlpb.ProviderListReply, error)
+	RequestDeviceScanStreamer(*ctlpb.DeviceScanRequest, *ctlpb.MgmtCtl_RequestDeviceScanStreamerServer) error
```

Comment:
the streaming should be encapsulated within the client method ideally and this streaming server not exported